server/routes/reporting/standard-report/utils.js

```diff
@@ -126,7 +37,11 @@ const constructPageHeading = (headingType, data) => {
   }
   const headingData = data[headingType];
 
-  return { title: headingData.displayName, data: headingDataMappers[headingData.dataType](headingData.value)};
+  if (headingData) {
+    return { title: headingData.displayName, data: headingDataMappers[headingData.dataType](headingData.value)};
+  }
+
+  return {};
```

Comment:
Need to handle not having a bottom border on the grid item if there isn't any data, probs just a little scss.

Thats more of a comment for the nunjuck file, but you didn't make any changes there,so feels relevant here.

Reply:
Put a fix in (put a &nbsp; in place of the div) and also updated the pdf generation to handle this as well.